src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1680,7 +1680,7 @@ namespace RCT1
                     dst2->SetStationIndex(StationIndex::FromUnderlying(src2->GetStationIndex()));
                     dst2->SetHasGreenLight(src2->HasGreenLight());
                     dst2->SetIsIndestructible(src2->IsIndestructible());
-                    if (rideType == RIDE_TYPE_GHOST_TRAIN)
+                    if (ride->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS))
```

Comment:
Only the Ghost Train had landscape doors, while OpenRCT2 could theoretically have it on more rides. I’d rather have this check for the RCT1 ride type instead.